gaia/gdal_functions.py

```diff
@@ -0,0 +1,402 @@
+import string
```

Comment:
@aashish24 There are GaiaProcesses that use these functions (SubsetProcess uses gdal_clip, RasterMathProcess uses gdal_calc).   I can move them to utils/geo.